error[E0271]: expected `[closure@fallback-closure-wrap.rs:18:40]` to be a closure that returns `()`, but it returns `!`
  --> $DIR/fallback-closure-wrap.rs:18:31
   |
LL |       let error = Closure::wrap(Box::new(move || {
   |  _______________________________^
LL | |
LL | |         panic!("Can't connect to server.");
LL | |     }) as Box<dyn FnMut()>);
   | |______^ expected `()`, found `!`
   |
   = note: expected unit type `()`
                   found type `!`
   = note: required for the cast from `[closure@$DIR/fallback-closure-wrap.rs:18:40: 18:47]` to the object type `dyn FnMut()`

error: aborting due to previous error

For more information about this error, try `crablangc --explain E0271`.
